examples/tutorials/core_api_mnist/model_def_checkpoints.py

```diff
@@ -0,0 +1,224 @@
+# Save and load to and from checkpoint storage. Introduce preemption support.
+
+from __future__ import print_function
+import argparse
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+from torchvision import datasets, transforms
+from torch.optim.lr_scheduler import StepLR
+import torch
+import determined as det
+import pathlib
```

Comment:
In model_def_metrics.py, you had an empty line above `import determined as det`.  I think that's helpful for our example, and I think you should keep it.

But `import pathlib` is a python standard library, and it shouldn't be down here.  It belongs between `import argparse` and the top `import torch`.

The normal ordering of python imports is usually something like:
* standard libraries, then 3rd party libraries, then your own local libraries
* with those categories, `import` statements come before `from x import y` statements
* within those categories, sort alphabetically

Though, it's hard to tell from their example if they're following the first rule, but they're definitely following the others.